src/svc_vc.c

```diff
@@ -661,6 +661,22 @@ svc_vc_stat(SVCXPRT *xprt)
 	return (XPRT_IDLE);
 }
 
+union proxy_addr {
```

Comment:
is there a reason why create our own structs and union here?

Reply:
(then again, I don't too much worry)

Reply:
OK, I've included an haproxy.h header with this stuff.